openedx/features/sdaia_features/course_progress/apps.py

```diff
@@ -1,19 +1,28 @@
 """
 Progress Updates App Config
 """
+
 from django.apps import AppConfig
 from edx_django_utils.plugins import PluginURLs, PluginSettings
 from openedx.core.djangoapps.plugins.constants import ProjectType, SettingsType
 
+
 class CourseProgressConfig(AppConfig):
-    name = 'openedx.features.sdaia_features.course_progress'
+    name = "openedx.features.sdaia_features.course_progress"
 
     plugin_app = {
+        "url_config": {
```

Comment:
Import config variables instead of using static strings.
Example:
```
PluginURLs.CONFIG: {
            ProjectType.LMS: {
                PluginURLs.NAMESPACE: '',
                PluginURLs.REGEX: fr'^courses/{COURSE_ID_PATTERN}/discussion/forum/',
                PluginURLs.RELATIVE_PATH: 'urls',
            }
        },
```